src/java/org/apache/cassandra/index/sai/disk/v1/PerColumnIndexFiles.java

```diff
@@ -71,6 +77,11 @@ public FileHandle balancedTree()
         return getFile(IndexComponent.BALANCED_TREE);
     }
 
+    public FileHandle vectors()
+    {
+        return getFile(IndexComponent.VECTOR);
```

Comment:
Maybe the index component should be named `VECTORS`, in the plural, the same as the method?